components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -182,6 +182,15 @@ open class GleanInternalAPI {
 
         // Set the OS type
         Baseline.os.set("Android")
+        // Set the OS version
+        // https://developer.android.com/reference/android/os/Build.VERSION
+        // TODO: We should probably send the SDK version as well
+        Baseline.osVersion.set(android.os.Build.VERSION.RELEASE)
```

Comment:
The legacy telemetry library for android uses [something different](https://github.com/mozilla-mobile/android-components/blob/9e2b38f8b98545baaee8d2dd5bb68dd4b9e6b98f/components/service/telemetry/src/main/java/org/mozilla/telemetry/measurement/OperatingSystemVersionMeasurement.java#L13): are they equivalent? If not, I think we should prefer using what the legacy library was sending for data equivalency.